examples/apps/whoami/README.md

```diff
@@ -0,0 +1,19 @@
+# whoami
+
+This example can be used to deploy the [whoami](https://hub.docker.com/r/containous/whoami) - a simple request and response logging service, similar to [httpbin](./../httpbin/README.md).
+
+## Deployment
+
+- To deploy the basic version of application, use the following command:
```

Comment:
```suggestion
- To deploy the basic version of the application, use the following command:
```
by application you mean whoami, correct?

Reply:
Yup.